app/helpers/application_helper.rb

```diff
@@ -29,6 +29,14 @@ def navigation_items
     []
   end
 
+  def show_confirmation_reminder?
+    return false unless current_user
+
+    !current_user.confirmed_at?
+  rescue Devise::MissingWarden
```

Comment:
This exception will only be triggered on the 429 page, so I don't think it can come up here.

Reply:
So I can just drop this line?

Reply:
Done here: https://github.com/alphagov/govuk-account-manager-prototype/pull/436/commits/c5fb6df330800ccb6d010681b29f3ee9c91bd6f9